src/Utils.ts

```diff
@@ -756,7 +779,7 @@ export const bigIntArrayToUnsignedInt = (
 };
 
 export function abvToArrayBuffer(buffer: ArrayBufferView): ArrayBuffer {
-  if (Buffer.isBuffer(buffer)) {
+  if (CraftzdogBuffer.isBuffer(buffer)) {
     return buffer.buffer;
```

Comment:
A similar issue will happen here, and it seems `randomFill` uses it

Reply:
if I change things around here to do the same thing as this PR does, then `randomFill` and friends' tests fail.  I'm not sure why yet.

Reply:
@tex0l check out #520 (first commit vs second commit).  If you change `abvToArrayBuffer` to `bufferLikeToArrayBuffer` in `random.ts`, nine of the 62 tests fail.  Thoughts?

Reply:
ok, left the non-copy function in there for `randomFill` and documented both.

Thanks for taking a look!